lib/tests/streamlit/runtime/runtime_test.py

```diff
@@ -75,18 +82,21 @@ class RuntimeSingletonTest(unittest.TestCase):
     def tearDown(self) -> None:
         Runtime._instance = None
 
+    @patch("streamlit.runtime.runtime.LocalSourcesWatcher", MagicMock())
```

Comment:
Since it seems like we need this for every test function, it might be better to move this to the testcase setup

Reply:
Ah - I didn't know you could do that! Handy